### PR TITLE
Add a script for bash configuration

### PR DESCRIPTION
**Pull request by JimmyOhn**
I add a script file for bash configuration. I think that this patch may help people to set easy configuration using it.

### CODE REVIEW

File: prerequisite.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ -e $HOME/.bashrc ]; then
+echo -e "
+if [ -n \"\$UNDER_JHBUILD\" ]; then
+  PS1=\"[jhbuild\$GST_VERSION] \$PS1\"
+fi " >> $HOME/.bashrc
+echo "JHbuild configuration completed successfully!"
```
Comment (joybro) [inline prerequisite.sh]: How about this? "JHBuild custom prompt is added successfully"

**Comment by joybro** [comment]
1. This is not prerequisite. setup-prompt.sh would be better.
2. Please modify README.sh to consider this.

**Comment by JimmyOhn** [comment]
I have modified above code for your comment. please review it.

**Comment by joybro** [comment]
Thank you for contribution!

**Pull request merged**